packages/@react-aria/overlays/src/calculatePosition.ts

```diff
@@ -411,3 +401,31 @@ export function calculatePosition(opts: PositionOpts): PositionResult {
     maxHeight
   );
 }
+
+function getOffset(node: Element): Offset {
+  let box: Offset = node.getBoundingClientRect();
+  let {scrollTop, scrollLeft, clientTop, clientLeft} = document.documentElement;
+  box.top += scrollTop - clientTop;
+  box.left += scrollLeft - clientLeft;
+  return box;
+}
```

Comment:
This errors when I attempt to open a Picker in the following story: https://reactspectrum.blob.core.windows.net/reactspectrum/7c18f4284ae7f6fd19e8b8be8da8ee28e82cc2b4/storybook/index.html?path=/story/picker--default&providerSwitcher-toastPosition=bottom. Similar thing happens which other overlay components (MenuTrigger, ComboBox, etc)